enhancements/rhcos/static-networking-enhancements.md

```diff
@@ -0,0 +1,101 @@
+
+### Goals
+  - Provide a guided configuration of networking and Ignition source information
+  - Provide tooling to create customized ISOs that include unique Ignition configurations
+  - Provide support for a VMWare "backchannel" to provide network configuration and Ignition source information
+
+## Proposal
+For the bare metal use case, we are proposing the use of the new [live ISO](https://github.com/openshift/enhancements/pull/210) to provide an interactive environment where users can configure their networking parameters and have them persist to the real root.  In this environment, users will be able to invoke a TUI that will allow them to configure networking and Ignition source.  This information will be used as part of the install of the host and will persist into the real root.
+
+Additionally, the new live ISO will provide the ability to embed the networking configuration via an Ignition config, resulting in a custom ISO that can be used to install to a host in an automated fashion.
+
+For the VMware use case, the proposal is to the existing `guestinfo` method to provide the static networking configuration.
+
+### User Stories
+
+#### Story 1
+A user wants to spin up a single FCOS node on bare metal using a static networking configuration.  They boot the live ISO and are dropped into an interactive FCOS environment.  They are able to use a tool to guide them to configure static networking for the host and provide a URL to the Ignition configuration.  The user runs `coreos-installer` after the configuration is complete and the host boots into the newly installed OS with static networking configured and Ignition config applied.
+
+### Story 2
+A customer has a fleet of bare metal machines they want to provision with RHCOS and static networking, that will later be used for the installation of OpenShift Container Platform (OCP).  They use some automation to generate unique Ignition configs that include the static networking configuration.  Using similar automation, they use `coreos-installer iso embed` to embed the Ignition configs into unique ISOs per machine.  These ISOs can then be attached to the baseboard machine controller (BMC) or lights out management (LOM) of the host, at which point the hosts can be booted and RHCOS will be automatically installed with the static networking configured and Ignition config applied.
+
+### Story 3
+A customer wants to deploy OCP onto VMware vSphere and use static networking for their hosts.  The do not want to use the bare metal ISO install path.  During the definition of the VM using the OVA image, the customer provides data to a `guestinfo` property that specifies the static networking configuration in the form of `dracut` kernel arguments.  When the VM is booted, the values in the `guestinfo` property are applied before networking is brought online and the VM continues to boot normally.
```

Comment:
nit: `s/The do not want to use the/For their specific environment it is impractical for them to use the/`